### PR TITLE
haha, merging master-to-master with new fixes must be fun

### PR DESCRIPTION
**Pull request by tinwelen**
i'm not an expert in git, so i just hope that the request is created correctly

### CODE REVIEW

**Comment by dangreen** [comment]
@tinwelen 🔥
@borzoni review plz 🙏

File: en/index.md [hunk 1]
```diff
@@ -214,13 +214,13 @@ else dsl(dsla[0],"/index.legacy.js");dsld.all[1].appendChild(dslf)</script>
...
 
 ## Dependencies transpilation
 
-Following aforesaid part of the article, we learned several ways of using Browserslist to reduce the size of the website's _own_ code, but the other part of the bundle is its dependencies. In a web applications, the size of the dependencies in the final bundle can take up a significant part.
+Following aforesaid part of the article, we've learned several ways of using Browserslist to reduce the size of the website's _own_ code, but the other part of the bundle is its dependencies. In web applications, the size of the dependencies in the final bundle can take up a significant part.
```
Comment (borzoni) [inline en/index.md:223]:
```suggestion
Following the aforesaid part of the article, we've learned several ways of using Browserslist to reduce the size of the website's _own_ code, but the other part of the bundle is its dependencies. In web applications, the size of the dependencies in the final bundle can take up a significant part.
```

File: en/index.md [hunk 2]
```diff
@@ -93,14 +93,14 @@ Thus, the website will send a lightweight bundle to users with new browsers, res
 
 #### Module/nomodule
 
-With browsers' support of ES-modules, there was found a way to implement DSL on the client side:
+With browsers' support of ES-modules, there was found a way to implement DSL on client side:
```
Comment (borzoni) [inline en/index.md:96]:
```suggestion
With browsers' support of ES-modules, there is a way to implement DSL on client side:
```

**Comment by dangreen** [comment]
@borzoni done?

**Comment by borzoni** [comment]
@dangreen yap, just need to fix the 7th line also. For other parts i've suggested some edits.

**Pull request merged**